### DISCUSSION TITLE
QA: Why Redis 6.2 and not Redis 7.X ?

### DISCUSSION THREAD

**Discussion by MickLesk** [immich-app/immich#10144] [Q&A] [answered]
Hello Dev's,

I've often had this question on my mind, so I thought I'd just ask.

Why is Redis 6.2.xx still being used? There are a few things in this version that don't run as smoothly as in 7.x.x. 

Redis 7.X also offers vector search, active-active databases, better ACLs and RBACs and possibly RediSearch 2.0

Or is the implementation to Redis 7 already in progress? :-)

**Comment by bo0tzz** [comment, marked as answer]
We'll be dropping redis entirely at some point, so no reason for us to upgrade now. It'll probably work fine if you run 7 though.

**Reply by MickLesk** [reply]
Sounds like a good plan :-)


Just a general question, I don't want to crash my installation, so I'll keep it that way and wait for the next releases :-)
